.changeset/selfish-lions-grin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vercel/node': patch
```

Comment:
Let's also update in `@vercel/next`, `@vercel/redwood` and `@vercel/remix`?

Reply:
Done!